src/main/webapp/app/entities/participation/participation.module.ts

```diff
@@ -16,24 +16,35 @@ import {
     ParticipationService,
     ParticipationWebsocketService,
 } from './';
+import { ParticipationSubmissionComponent } from 'app/entities/participation-submission/participation-submission.component';
 import { SortByModule } from 'app/components/pipes';
 import { ArtemisProgrammingExerciseModule } from 'app/entities/programming-exercise/programming-exercise.module';
+import { ArtemisResultModule } from 'app/entities/result';
+import {
+    ParticipationSubmissionDeleteDialogComponent,
+    ParticipationSubmissionDeletePopupComponent,
+} from 'app/entities/participation-submission/participation-submission-delete-dialog.component';
 
 const ENTITY_STATES = [...participationRoute, ...participationPopupRoute];
 
 @NgModule({
-    imports: [ArtemisSharedModule, RouterModule.forChild(ENTITY_STATES), SortByModule, ArtemisProgrammingExerciseModule],
+    imports: [ArtemisSharedModule, RouterModule.forChild(ENTITY_STATES), SortByModule, ArtemisProgrammingExerciseModule, ArtemisResultModule],
     declarations: [
         ParticipationComponent,
         ParticipationDeleteDialogComponent,
         ParticipationDeletePopupComponent,
+        ParticipationSubmissionDeleteDialogComponent,
+        ParticipationSubmissionDeletePopupComponent,
         ParticipationCleanupBuildPlanDialogComponent,
         ParticipationCleanupBuildPlanPopupComponent,
+        ParticipationSubmissionComponent,
```

Comment:
Ideally the 3 components should be declared in their own module (e.g. participation-submission.module.ts), which is then imported by the participation.module.ts.
It should be positioned within the participation.module's folder structure. 

We currently have some modules that are already pretty big which makes it harder to reason about them.